tests/integration/test_parallel_analytics_config_batch4.py

```
@@ -0,0 +1,2332 @@
+        import asyncio
+        from file_organizer.api.service_facade import ServiceFacade
+
+        facade = ServiceFacade()
+
+        with patch(
+            "file_organizer.services.deduplication.detector.DuplicateDetector"
+        ) as mock_cls:
+            mock_cls.side_effect = RuntimeError("dedup broken")
+
+            async def run() -> dict[str, Any]:
+                return await facade.find_duplicates("/tmp/scan")
```

Ruff (0.15.9)

[error] Probable insecure usage of temporary file or directory: "/tmp/scan"

(S108)